api/jobs_test.go

```
@@ -78,12 +84,14 @@ func expectedJob(id string,
 		State:                        state,
 		TotalSearchDocuments:         totalSearchDocuments,
 		TotalInsertedSearchDocuments: totalInsertedSearchDocuments,
-	}
+	}, nil
```

By doing COMMENT 2, you don't need to return an error